client/my-sites/stats/activity-log-day/style.scss

```diff
@@ -73,6 +73,7 @@
 
 .activity-log-day__day {
 	font-weight: 600;
+	white-space: nowrap;
```

Comment:
I don't think the days with the events overflow because they aren't in ranges (that I'm aware of). Is this needed?

Reply:
the CSS class is used in both individual days and date ranges